### ISSUE TITLE
Break up classes into separate files - module perhaps

### ISSUE THREAD

**Issue by sgtnasty** [nnasa14/recipes#6]
Use the standard python module [layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/)

```
root/
    /recipes
    /recipes/__init__.py
    /recipes/__main__.py
    /recipes/recipes.py
    /recipes/ingredients.py
```

**Comment by sgtnasty** [comment]
Using the example above `root` would be `~/Projects/recipes/src` if using the layout described.
**Otherwise** is would be: `~/Projects/recipes`

**Comment by nnasa14** [comment]
Issue closed with commits 0b9f57857443a619f8039eba4b920167bd182573 and c820379edf3111b302c95f5e5fa7ec06614d3a2b
